aiida/README.md

```diff
@@ -4,34 +4,28 @@
 {% endif %}
 
 {% if archive_url %}
-This project imports an [AiiDA archive]({{ archive_url }}) upon starting an environment, which may take some time.
+This project comes with a Jupyter notebook for importing and exploring an [AiiDA archive]({{ archive_url }}).
 {% endif %}
 
-
 ### Introduction
 
-This project comes with an AiiDA environment already pre-configured for your user.
+This project comes with a pre-configured AiiDA environment:
 
  1. Start a new environment (e.g. 1 CPU, 2GB memory)
  1. Open the terminal and type
     ```
-    verdi status
-    ```
-
-You should see a status similar to
-```
- ✔ config dir:  /work/{{ __sanitized_project_name__ }}/repo/.aiida
- ✔ profile:     On profile {{ __sanitized_project_name__ }}
- ✔ repository:  /work/{{ __sanitized_project_name__ }}/repo/.aiida/repository/{{ __sanitized_project_name__ }}
- ✔ postgres:    Connected as aiidauser@localhost:5432
- ✔ rabbitmq:    Connected as amqp://guest:guest@127.0.0.1:5672?heartbeat=600
- ✔ daemon:      Daemon is running as PID 30575 since 2020-11-18 11:15:30
- ```
+    $ verdi status
+     ✔ config dir:  /work/{{ __sanitized_project_name__ }}/repo/.aiida
+     ✔ profile:     On profile default
+     ✔ repository:  /work/{{ __sanitized_project_name__ }}/repo/.aiida/repository/default
+     ✔ postgres:    Connected as aiidauser@localhost:5432
+     ✔ rabbitmq:    Connected as amqp://guest:guest@127.0.0.1:5672?heartbeat=600
+     o daemon:      The daemon is not running
+     ```
+ 1. Open the Jupyter notebook in the `notebooks/` folder or see below for how to get started with AiiDA.
```

Comment:
The rendering is slightly off in this section on RenkuLab because the <code>```</code> should be at the beginning of the line
![Screenshot_20210504_141033](https://user-images.githubusercontent.com/43481553/117003630-5e07f480-ace5-11eb-9484-0afceb4986aa.png)
This doesn't happen on the preview here, I guess GitHub has a better renderer for markdown files :shrug: 

Moving <code>```</code> at the beginning of the line fixes it. Notice that I changed also the numbers because the last one would be considered a new numbered list since now there is unindented code between the second and third bullet point.

<pre>
 2. Open the terminal and type
```
    $ verdi status
     ✔ config dir:  /work/{{ __sanitized_project_name__ }}/repo/.aiida
     ✔ profile:     On profile default
     ✔ repository:  /work/{{ __sanitized_project_name__ }}/repo/.aiida/repository/default
     ✔ postgres:    Connected as aiidauser@localhost:5432
     ✔ rabbitmq:    Connected as amqp://guest:guest@127.0.0.1:5672?heartbeat=600
     o daemon:      The daemon is not running
```
 3. Open the Jupyter notebook in the `notebooks/` folder or see below for how to get started with AiiDA.
</pre>

Reply:
thanks!